### ISSUE TITLE
how to test and evaluate it ?

### ISSUE THREAD

**Issue by cuimiao187561** [qqwweee/keras-yolo3#439]
Is it possible to evaluate our model on thousands of images(test.txt and val.txt)? How to get the loss, recall, accuracy, average accuracy and mAP indicator results.

**Comment by bingoxumo** [comment]
i use the faster rcnn voc eval to evaluate

**Comment by zizzizziz** [comment]
> i use the faster rcnn voc eval to evaluate

How to use, Could share to me?

**Comment by dek8v5** [comment]
yes, please share

**Comment by mohong96** [comment]
> i use the faster rcnn voc eval to evaluate

please share,thx

**Comment by casstio** [comment]
> i use the faster rcnn voc eval to evaluate

can you please share it to me?thx very much

**Comment by zhangfx123** [comment]
可以分享一下吗，谢谢

**Comment by Lanme** [comment]
`import yolo
from PIL import Image
import matplotlib.pyplot as plt
%matplotlib inline
model = yolo.YOLO(model_path='logs/ep099-loss6.623-val_loss6.542.h5',
        anchors_path='model_data/yolo_anchors.txt',
        classes_path= 'model_data/voc_classes.txt',
        score =0.3,
        iou = 0.45,
        model_image_size =(416, 416),
        gpu_num = 1)

images = Image.open("data/1.jpg")
result = model.detect_image(images)
plt.imshow(result)
`
